src/ReceiveSharingIntent.ts

```diff
@@ -45,7 +45,10 @@ class ReceiveSharingIntentModule implements IReceiveSharingIntent {
   clearReceivedFiles() {
     // https://github.com/ajith-ab/react-native-receive-sharing-intent/issues/149
     // this.isClear = true;
-    ReceiveSharingIntent.clearFileNames();
+
+    if (!this.isIos) {
```

Comment:
Could you please add comment why we don't do this for iOS?

Reply:
Added as a todo comment because I'm hopeful it can be fixed to work properly 😅